### PR TITLE
feat: Add openCamera web support

### PR DESCRIPTION
**Pull request by jardelbordignon**
To test the functionality on the web just create a react-native-web app and use the openCamera function as per the official documentation

### CODE REVIEW

**Review by mateoguzmana — COMMENTED** [review]
Great work on this @jardelbordignon! great to see some effort to increase web support 🚀

How does this look like? Can we add some screenshots in the PR or a usage example in the [example app](https://github.com/react-native-image-picker/react-native-image-picker/tree/main/example)? (not sure if there are changes needed to make it work in the example app)

**Comment by jardelbordignon** [comment]
Hey! I appreciate the work of maintaining the project. The way to use it is
exactly the same, just follow the official documentation. To test. I made
an example of use in this link https://health-gym.onrender.com/ login with user
-> ***@***.*** password -> Pwd123! after login click on Profile link

Em sáb., 8 de abr. de 2023 às 19:47, Mateo Guzmán ***@***.***>
escreveu:

File: src/platforms/NativeImagePicker.web.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+/* eslint-disable @typescript-eslint/ban-types */
```
Comment (mateoguzmana) [inline src/platforms/NativeImagePicker.web.ts:1]:
```suggestion
```

Is this needed? I removed it locally and TS transpiles correctly without errors

**Review by mateoguzmana — APPROVED** [review]
I tested it locally with the example app and it works like a charm. Amazing work!

File: src/platforms/NativeImagePicker.web.ts (hunk 1 above)
Comment (jardelbordignon) [reply src/platforms/NativeImagePicker.web.ts:1]: This was to suit my lint setup, but it can be removed.

**Comment by NeriRos** [comment]
Is this ready to merge?

**Review by MrMole96 — APPROVED** [review]

**Review by Johan-dutoit — APPROVED** [review]

**Pull request merged**